packages/remark-math/package-lock.json

```diff
@@ -1,11 +1,1596 @@
 {
-	"requires": true,
+	"name": "@ibrahima/remark-math",
```

Comment:
Probably will end up redoing adding packages to ensure that package-lock.json has the right stuff.